integration/boofcv-WebcamCapture/examples/boofcv/examples/ExampleTrackingKltGst.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2011-2020, Peter Abeles. All Rights Reserved.
+ *
+ * This file is part of BoofCV (http://boofcv.org).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package boofcv.examples;
+
+import boofcv.abst.feature.detect.interest.ConfigPointDetector;
+import boofcv.abst.feature.detect.interest.PointDetectorTypes;
+import boofcv.abst.tracker.PointTrack;
+import boofcv.abst.tracker.PointTracker;
+import boofcv.alg.tracker.klt.ConfigPKlt;
+import boofcv.factory.tracker.FactoryPointTracker;
+import boofcv.gui.feature.VisualizeFeatures;
+import boofcv.gui.image.ImagePanel;
+import boofcv.gui.image.ShowImages;
+import boofcv.io.gstwebcamcapture.UtilWebcamCapture;
+import boofcv.io.gstwebcamcapture.Webcam;
+import boofcv.io.image.ConvertBufferedImage;
+import boofcv.struct.image.GrayF32;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.util.List;
+
+/**
+ * Processes a video feed and tracks points using KLT
+ *
+ * @author Peter Abeles
+ */
+public class ExampleTrackingKltGst {
```

Comment:
My vote is to remove most of these examples that show more "advanced image processing" since we only need one example that shows how to use the API, unless there are other aspects of gstreamer you wish to show off.